client/me/concierge/shared/no-available-times.js

```diff
@@ -12,35 +12,51 @@ class NoAvailableTimes extends Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { translate, isUserBlocked } = this.props;
 		return (
 			<div>
 				<PrimaryHeader />
 				<Card>
 					<h2 className="shared__no-available-times-heading">
 						{ translate( 'Sorry, all upcoming sessions are full.' ) }
 					</h2>
-					{ translate(
-						'We add new sessions daily, so please check back soon for more options. In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site. {{externalLink1}}View webinars{{/externalLink1}} or {{externalLink2}}contact us in Live Chat{{/externalLink2}}.',
-						{
-							components: {
-								externalLink1: (
-									<ExternalLinkWithTracking
-										icon={ false }
-										href="/webinars"
-										tracksEventName="calypso_concierge_book_view_webinars"
-									/>
-								),
-								externalLink2: (
-									<ExternalLinkWithTracking
-										icon={ false }
-										href="/help/contact"
-										tracksEventName="calypso_concierge_book_contact_us"
-									/>
-								),
-							},
-						}
-					) }
+					{ isUserBlocked &&
+						translate(
+							'We add new sessions daily, so please check back soon for more options. In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site. {{externalLink1}}View webinars{{/externalLink1}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'If you don’t see a day or time that works for you, please check back soon for more options! In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site. {{externalLink}}View webinars{{/externalLink}}.' )` *ES Score: 6*